Depend-on/Bash.sh

```diff
@@ -0,0 +1,221 @@
+                    echo "freerdp2-devel"
+                    echo "pango-devel"
+                    echo "libssh2-devel"
+                    echo "libtelnet-devel"
+                    echo "libvncserver-devel"
+                    echo "libwebsockets-devel"
+                    echo "pulseaudio-libs-devel"
+                    echo "openssl-devel"
+                    echo "libvorbis-devel"
+                    echo "libwebp-devel"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo zypper install -y FFmpeg
+                            sudo zypper install -y libavcodec-devel libavformat-devel libavutil-devel libswscale-devel
+                            sudo zypper install -y freerdp2-devel
+                            sudo zypper install -y pango-devel
+                            sudo zypper install -y libssh2-devel
+                            sudo zypper install -y libtelnet-devel
+                            sudo zypper install -y libvncserver-devel
+                            sudo zypper install -y libwebsockets-devel
+                            sudo zypper install -y pulseaudio-libs-devel
+                            sudo zypper install -y openssl-devel
+                            sudo zypper install -y libvorbis-devel
+                            sudo zypper install -y libwebp-devel
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                elif command -v pkg &>/dev/null; then
+                    echo "The following packages will be installed:"
+                    echo "FFmpeg"
+                    echo "libavcodec-devel libavformat-devel libavutil-devel libswscale-devel"
+                    echo "freerdp2-devel"
+                    echo "pango-devel"
+                    echo "libssh2-devel"
+                    echo "libtelnet-devel"
+                    echo "libvncserver-devel"
+                    echo "libwebsockets-devel"
+                    echo "pulseaudio-libs-devel"
+                    echo "openssl-devel"
+                    echo "libvorbis-devel"
+                    echo "libwebp-devel"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo pkg install -y <package>
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                else
+                    echo "Unsupported Linux distribution."
+                fi
+                ;;
+        esac
+    else
+        echo "Unsupported operating system."
+    fi
+}
+
+identify_and_run
+
+
+#done
+#Rick Astley
```

Comment:
We try not Rick-roll people as part of the source code ;-).

Reply:
I will put his video there nah when you say no I will follow that to the T